t/op/loopctl.t

```diff
@@ -922,6 +922,23 @@ TEST41: {
 }
 cmp_ok($ok,'==',1,'dynamically scoped');
 
```

Comment:
Should we include a comment at this point along the lines of "Fixes https://github.com/Perl/perl5/issues/18369"?

Reply:
Done